CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ All notable changes to this project will be documented in this file.
 * Fix:
 	* Retain cycle in example project
 
+## [1.3.0]
+* Feature:
```

Comment:
this should be on the [X.X.X] section for the next release. 
version number depends on other features.

Reply:
Done

Reply:
Remove [1.3.0] and move Feature in the [X.X.X] section.

Reply:
Done